### ISSUE TITLE
https://www.youtube.com/tv  will close soon

### ISSUE THREAD

**Issue by luckiers** [yuliskov/SmartYouTubeTV#340]
How to treat this problem？This app is web based。

**Comment by Divya015** [comment]
Yes this app is web based, how can we support the app on our device. Please suggest

**Comment by rodalpho** [comment]
Definitely concerned about this, will this change break SYTV?

**Comment by yuliskov** [comment]
Definitely yes. But what about WebOS and Tizen powered devices?

2019-09-24 4:18 GMT+03:00, rodalpho <notifications@github.com>:

**Comment by rodalpho** [comment]
Oof. That's heartbreaking.

How do they work behind the scenes? All the 10 foot UI YouTube apps look identical to  www.youtube.com/TV including Google's own AndroidTV. Perhaps the website will still exist, but require some sort of authentication or token to access?

**Comment by jnines** [comment]
It's based on user agent, there's a list of them here.
https://developers.whatismybrowser.com/useragents/explore/operating_platform/smart-tv/
I use the first Tizen one on my WMC box, don't get the warning anymore.

**Comment by rodalpho** [comment]
Well that explains why SYTV still works, when the site was retired 4 days ago! 

Phew.

**Comment by MaxHart828** [comment]
I saw this... 

"Of course, the removal of the YouTube leanback web app shouldn’t affect the official YouTube apps which use the same interface. Android TV, Roku, and other officially supported TV platforms use this interface and if anything, disabling the web app should make those a bit more flexible to new updates." here https://9to5google.com/2019/09/16/youtube-leanback-tv-interface-disabled-soon/ 

Does that mean SYTV will also continue to function?

**Comment by jnines** [comment]
The removal of the base leanback was meant to hurt htpc users using adblockers, the tv/set top box/stick users won't be affected.  So long as SYTV uses one of their user agents it should work.

**Comment by rodalpho** [comment]
Ahhh so it just checks the user-agent, they didn't completely retire the site. 

I mean, why bother? Amazon would have just faked a different user-agent, if they had to. I guess this change would counter Amazon's argument that they were "just accessing a website on the internet" with their web frontend, when made to people lacking technical sophistication like a judge and jury.

**Comment by MaxHart828** [comment]
> The removal of the base leanback was meant to hurt htpc users using adblockers, the tv/set top box/stick users won't be affected. So long as SYTV uses one of their user agents it should work.

Okay, thanks. Would hate it if SYTV stopped working. Much better than the official app.
